docs/_templates/index.html

```diff
@@ -0,0 +1,302 @@
+                    <img class="rounded-circle rounded-header-image"
+                        src="https://docs.pytest.org/en/7.2.x/_static/pytest_logo_curves.svg"
+                        alt="pytest logo" width="140" height="140">
+                    <h2>Prevent regressions</h2>
+                    <p>
+                        You can easily integrate PyStack into your
+                        <code>pytest</code> test suite. If a test gets stuck,
+                        our plugin can automatically report each thread's stack
+                        and help you figure out what went wrong.
+                    </p>
+                    <p><a class="btn btn-secondary filled-pystack"
+                        href="https://github.com/bloomberg/pytest-pystack"
+                        role="button">Discover the plugin &raquo;</a></p>
+                </div><!-- /.col-lg-4 -->
+            </div><!-- /.row -->
+
+            <!-- START THE FEATURETTES -->
+            <hr class="featurette-divider">
+
+            <div class="row featurette">
+                <div class="col-md-7">
+                    <h2 class="featurette-heading">
+                        Uncover your program's secrets.
+                        <span class="text-muted">No more deadlocks or mysterious crashes.</span>
+                    </h2>
+                    <p class="lead">
+                        PyStack provides a powerful tool for inspecting the
+                        stack frames of a running Python process or a Python
+                        core dump. It allows developers to quickly and easily
+                        understand what's happening in their Python programs
+                        without delving into the intricate details of CPython
+                        internals. When you ask PyStack to analyze a running
+                        process or core file, it prints a stack trace for each
+                        thread, which shows the sequence of function calls and
+                        their corresponding line numbers in the Python code.
+                        This feature is invaluable for debugging and
+                        understanding the runtime behavior of an application,
+                        especially when encountering unexpected hangs or
+                        crashes.
+                    </p>
+                </div>
+                <div class="col-md-5">
+                    <img class="featurette-image img-fluid mx-auto reveal_l"
+                        src="{{ pathto('_static/images/stack.png', 1) }}"
+                        alt="Python traceback">
+                </div>
+            </div>
+
+            <hr class="featurette-divider">
+
+            <div class="row featurette">
+                <div class="col-md-7 order-md-2">
+                    <h2 class="featurette-heading">
+                        No more hidden code!
+                        <span class="text-muted">See into native extensions.</span>
```

Comment:
Perhaps a better synonym for "See", like "Peer into.." or "Dive into"

Reply:
We have "Deep Dive into Program State." below... Let's go with:
```suggestion
                        <span class="text-muted">Peer inside native extensions.</span>
```